GameDataParser/Parsers/ItemParser.cs

```diff
@@ -206,6 +208,12 @@ protected override List<ItemMetadata> Parse()
                         }
                     }
                 }
+                if (contentType == "ChatEmoticonAdd")
+                {
+                    string rawParameter = function.Attributes["parameter"].Value;
+                    int parameter = int.Parse((rawParameter.Where(x => char.IsDigit(x)).ToArray())); // grab the proper value from parameter attribute
```

Comment:
What `.ToArray()` does here?

Reply:
Not really sure how else I would grab the value.

Reply:
I think the correct solution here is a regex expression. But I managed the desired results with:
`int parameter = int.Parse(rawParameter.FirstOrDefault(x => char.IsDigit(x)).ToString());`
Results:
https://gist.github.com/angelotadeucci/deb88b118a2e5b7503f39d02ef0855f1

Reply:
Egh...looks like the expiration date is in some of them.